api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1991,14 +1991,16 @@ type RequiredAddressSet struct {
 	NonTLS bool `json:"nonTLS,omitempty"`
 }
 
+type CrashLoopList map[string](map[string]None)
+
 // BuggifyConfig provides options for injecting faults into a cluster for testing.
 type BuggifyConfig struct {
 	// NoSchedule defines a list of process group IDs that should fail to schedule.
 	NoSchedule []string `json:"noSchedule,omitempty"`
 
 	// CrashLoops defines a list of process group IDs that should be put into a
 	// crash looping state.
-	CrashLoop []string `json:"crashLoop,omitempty"`
+	CrashLoop CrashLoopList `json:"crashLoop,omitempty"`
```

Comment:
All changes to v1beta1 can be revert. We don't change those files and they are only there until we remove the v1beta1 support from our CRD.